string id
string type
string name
string date
string path